src/ice.c

```diff
@@ -3997,6 +3997,20 @@ static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data
 	}
 
 	if(pc && pc->do_transport_wide_cc && medium) {
+		/* Simulcast layers that have not received data will not have a recorded peer SSRC. */
+		/* Pick the first layer that has a peer SSRC */
+		guint32 ssrc_peer = 0;
+		for(int i = 0; i < 3; i++) {
```

Comment:
Per code style, we don't define variables inside the `for` loop, so please define `i` outside.